gravity/state.py

```diff
@@ -389,16 +389,25 @@ class GalaxyGxItProxyService(Service):
         "forward_ip": "--forwardIP {settings[forward_ip]}",
         "forward_port": "--forwardPort {settings[forward_port]}",
         "reverse_proxy": "--reverseProxy",
+        "proxy_path_prefix": "--proxyPathPrefix {settings[proxy_path_prefix]}",
     }
     _command_template = "{virtualenv_bin}npx gx-it-proxy --ip {settings[ip]} --port {settings[port]}" \
                         " --sessions {settings[sessions]} {command_arguments[verbose]}" \
                         " {command_arguments[forward_ip]} {command_arguments[forward_port]}" \
-                        " {command_arguments[reverse_proxy]}"
+                        " {command_arguments[reverse_proxy]} {command_arguments[proxy_path_prefix]}"
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         # override from Galaxy config if set
         self.settings["sessions"] = self.config.app_config.get("interactivetools_map", self.settings["sessions"])
+        # this can only be set in Galaxy config
+        it_base_path = self.config.app_config.get("interactivetools_base_path")
+        if it_base_path is not None:
```

Comment:
@natefoo I don't think it makes sense to test whether `interactivetools_base_path` is set, as the default value in any case is `/`. Setting `interactivetools_base_path` to `None` is also not checked in the Galaxy interactive tool code and will also break for interactive tools that have set `requires_domain` to False.

Reply:
Right, I see and agree! This also what we did in the other PR: https://github.com/galaxyproject/gravity/blob/c0e37fdedfc6df6835f2effdc735b8d63385a9a1/gravity/config_manager.py#L182

I think the `it_base_path` and `it_prefix` declarations should then be moved together, e.g.:

```
it_base_path = self.config.app_config.get("interactivetools_base_path", "/")
it_prefix = self.config.app_config.get("interactivetools_prefix", "interactivetool")
```